### PR TITLE
Add unit test for GRPCExporter and GRPCExporterProvider

### PR DESCRIPTION
**Pull request by flycash**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance
- [X] Add tests

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

**Comment by wu-sheng** [comment]
Ref to #2438

File: oap-server/exporter/pom.xml [hunk 1]
```diff
@@ -35,6 +35,10 @@
             <artifactId>server-core</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-testing</artifactId>
```
Comment (wu-sheng) [inline oap-server/exporter/pom.xml]: I think you miss dependency scope. This is a test dependency.

File: oap-server/exporter/src/test/java/org/apache/skywalking/oap/server/exporter/provider/grpc/MockLongValueIndicator.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ */
+
+package org.apache.skywalking.oap.server.exporter.provider.grpc;
+
+import org.apache.skywalking.oap.server.core.analysis.indicator.IntValueHolder;
+
+/**
+ * Created by dengming, 2019.04.20
+ */
+public class MockLongValueIndicator extends MockIndicator implements IntValueHolder {
```
Comment (wu-sheng) [inline oap-server/exporter/src/test/java/org/apache/skywalking/oap/server/exporter/provider/grpc/MockLongValueIndicator.java]: A Mock Long value indicator implement IntValueHolder? Is this true?

File: oap-server/exporter/src/test/java/org/apache/skywalking/oap/server/exporter/provider/grpc/MockIntValueIndicatior.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ */
+
+package org.apache.skywalking.oap.server.exporter.provider.grpc;
+
+import org.apache.skywalking.oap.server.core.analysis.indicator.LongValueHolder;
+
+/**
+ * Created by dengming, 2019.04.20
+ */
+public class MockIntValueIndicatior extends MockIndicator implements LongValueHolder {
```
Comment (wu-sheng) [inline oap-server/exporter/src/test/java/org/apache/skywalking/oap/server/exporter/provider/grpc/MockIntValueIndicatior.java]: A Mock Int value indicator implement LongValueHolder? Is this true?

File: oap-server/exporter/pom.xml (hunk 1 above)
Comment (flycash) [reply oap-server/exporter/pom.xml]: I found the scope was declared as test in the parent pom

File: oap-server/exporter/src/test/java/org/apache/skywalking/oap/server/exporter/provider/grpc/MockIntValueIndicatior.java (hunk 3 above)
Comment (flycash) [reply oap-server/exporter/src/test/java/org/apache/skywalking/oap/server/exporter/provider/grpc/MockIntValueIndicatior.java]: It was a mistake and now I fixed it.

File: oap-server/exporter/pom.xml (hunk 1 above)
Comment (wu-sheng) [reply oap-server/exporter/pom.xml]: Yes, this should be fine

**Comment by wu-sheng** [comment]
CI fails.

**Pull request merged**